src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -94,10 +94,10 @@ BookmarkManager.defaultBookmarks=Default bookmarks
 BookmarkManager.list=List of bookmarks
 BookmarkManager.listLong=A list of bookmarked freesites
 BookmarkManager.malformedBookmark=Malformed Bookmark
-Bookmarks.Defaults.Name.categoryIndexes=Directories of websites on Freenet
+Bookmarks.Defaults.Name.categoryIndexes=Directories of Freenet sites
 Bookmarks.Defaults.Name.categoryDocumentation=Freenet related documentation
-Bookmarks.Defaults.Name.categorySoftware=Freenet related software
-Bookmarks.Defaults.Name.categoryBlogs=Freenet team's blogs
+Bookmarks.Defaults.Name.categorySoftware=Software related to Freenet
+Bookmarks.Defaults.Name.categoryBlogs=Freenet development blogs
```

Comment:
Agreed, most of them are personal blogs which are often but not always development related.